docs/dashboard/configuration/channels.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Channels
+---
+
+This page allows you to define and manage your sales channels for your store. You can create multiple channels with different currencies.
+
+![Channels](../screenshots/channels-main.jpg)
+
+## How to
+
+### How to create a channel
+
+1. The first step is to click the _CREATE CHANNEL_ button to create a channel in the _Channels_ section of dashboard configuration.
+
+2. The second step is to fill out the following information:
+   - Channel name
+   - Slug
+   - Channel currency
+
+   Once you have provided the above information, click the _SAVE_ button to save the channel.
+
+![New channel](../screenshots/channels-create-new.jpg)
+
+### How to activate a channel
+
+Once a channel has been created, it will be inactive by default. Click the _ACTIVATE_ button to activate the channel. Once the channel has been activated, the status will be changed to _ACTIVE_. To deactive the channel, click the _DEACTIVE_ button.
+
+![New channel](../screenshots/channels-status.jpg)
+
+### How to remove a channel
+
+You can remove an individual channel from the channels list using the trash icon next to the channel.
```

Comment:
Maybe we should add here information about restrictions when users try to remove a channel?

Reply:
What restrictions? @fowczarek